docs/encrypted_storage.md

```diff
@@ -0,0 +1,51 @@
+# Aries Encrypted Storage
+
+The Aries framework can be figured to use encrypted storage. A concrete [EDV](https://identity.foundation/confidential-storage/) implementation is provided, but custom encrypted storage solutions can be injected.
```

Comment:
s/implementation/client/

Reply:
yes - should clarify that a client implementation is provided.

Reply:
Changed the wording. Take a look and let me know if it sounds good.